src/content/core/components/namespaces.mdx

```diff
@@ -31,18 +46,27 @@ To share a namespace go to the Okteto dashboard, select the namespace you want t
 
 <p align="center"><Image src={require('@site/static/img/share.png').default} alt="share a namespace" width="850" /></p>
 
-In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto dashboardll update the necessary permissions and notify your teammates via email.
+In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto will update the necessary permissions and notify your teammates via email.
 
-You can sleep your namespace from the command line with `okteto namespace sleep [name]`.
 
-When you want to wake your namespace, you can use the command `okteto namespace wake [name]`.
+## Transfer namespace ownership
+
+Okteto administrators have the ability to transfer non-default namespaces between users. This process can be achieved through the Okteto UI.
```

Comment:
Is there somewhere we can link `non-default` namespaces to? Not sure this is a widely understood concept for users/admins

Reply:
Yes, agreed. I feel like information on default namespaces should be covered in this page. The namespace management section at the top really needs an overhaul. I'm going to make a comment to handle that work separate from this PR.